src/components/tile-details-panel/texture-section.tsx

```diff
@@ -0,0 +1,418 @@
+import styled from "styled-components";
+import { Title, TileInfoSectionWrapper } from "../common";
+import { useEffect, useState } from "react";
+import { Tile3D } from "@loaders.gl/tiles";
+import { ModalDialog } from "../modal-dialog/modal-dialog";
+import {
+  getSupportedGPUTextureFormats,
+  GL_EXTENSIONS_CONSTANTS,
+} from "@loaders.gl/textures";
+import { Texture2D, instrumentGLContext, Program } from "@luma.gl/core";
+
+const {
+  COMPRESSED_RGB_S3TC_DXT1_EXT,
+  COMPRESSED_RGBA_S3TC_DXT1_EXT,
+  COMPRESSED_RGBA_S3TC_DXT3_EXT,
+  COMPRESSED_RGBA_S3TC_DXT5_EXT,
+  COMPRESSED_RGB_PVRTC_4BPPV1_IMG,
+  COMPRESSED_RGBA_PVRTC_4BPPV1_IMG,
+  COMPRESSED_RGB_PVRTC_2BPPV1_IMG,
+  COMPRESSED_RGBA_PVRTC_2BPPV1_IMG,
+  COMPRESSED_RGB_ATC_WEBGL,
+  COMPRESSED_RGBA_ATC_EXPLICIT_ALPHA_WEBGL,
+  COMPRESSED_RGBA_ATC_INTERPOLATED_ALPHA_WEBGL,
+  COMPRESSED_RGB_ETC1_WEBGL,
+  COMPRESSED_RGBA_ASTC_4X4_KHR,
+  COMPRESSED_RGBA_ASTC_5X4_KHR,
+  COMPRESSED_RGBA_ASTC_5X5_KHR,
+  COMPRESSED_RGBA_ASTC_6X5_KHR,
+  COMPRESSED_RGBA_ASTC_6X6_KHR,
+  COMPRESSED_RGBA_ASTC_8X5_KHR,
+  COMPRESSED_RGBA_ASTC_8X6_KHR,
+  COMPRESSED_RGBA_ASTC_8X8_KHR,
+  COMPRESSED_RGBA_ASTC_10X5_KHR,
+  COMPRESSED_RGBA_ASTC_10X6_KHR,
+  COMPRESSED_RGBA_ASTC_10X8_KHR,
+  COMPRESSED_RGBA_ASTC_10X10_KHR,
+  COMPRESSED_RGBA_ASTC_12X10_KHR,
+  COMPRESSED_RGBA_ASTC_12X12_KHR,
+  COMPRESSED_SRGB8_ALPHA8_ASTC_4X4_KHR,
+  COMPRESSED_SRGB8_ALPHA8_ASTC_5X4_KHR,
+  COMPRESSED_SRGB8_ALPHA8_ASTC_5X5_KHR,
+  COMPRESSED_SRGB8_ALPHA8_ASTC_6X5_KHR,
+  COMPRESSED_SRGB8_ALPHA8_ASTC_6X6_KHR,
+  COMPRESSED_SRGB8_ALPHA8_ASTC_8X5_KHR,
+  COMPRESSED_SRGB8_ALPHA8_ASTC_8X6_KHR,
+  COMPRESSED_SRGB8_ALPHA8_ASTC_8X8_KHR,
+  COMPRESSED_SRGB8_ALPHA8_ASTC_10X5_KHR,
+  COMPRESSED_SRGB8_ALPHA8_ASTC_10X6_KHR,
+  COMPRESSED_SRGB8_ALPHA8_ASTC_10X8_KHR,
+  COMPRESSED_SRGB8_ALPHA8_ASTC_10X10_KHR,
+  COMPRESSED_SRGB8_ALPHA8_ASTC_12X10_KHR,
+  COMPRESSED_SRGB8_ALPHA8_ASTC_12X12_KHR,
+  COMPRESSED_R11_EAC,
+  COMPRESSED_RG11_EAC,
+  COMPRESSED_SIGNED_RG11_EAC,
+  COMPRESSED_RGB8_ETC2,
+  COMPRESSED_RGBA8_ETC2_EAC,
+  COMPRESSED_SRGB8_ETC2,
+  COMPRESSED_SRGB8_ALPHA8_ETC2_EAC,
+  COMPRESSED_RGB8_PUNCHTHROUGH_ALPHA1_ETC2,
+  COMPRESSED_SRGB8_PUNCHTHROUGH_ALPHA1_ETC2,
+  COMPRESSED_RED_RGTC1_EXT,
+  COMPRESSED_SIGNED_RED_RGTC1_EXT,
+  COMPRESSED_RED_GREEN_RGTC2_EXT,
+  COMPRESSED_SIGNED_RED_GREEN_RGTC2_EXT,
+  COMPRESSED_SRGB_S3TC_DXT1_EXT,
+  COMPRESSED_SRGB_ALPHA_S3TC_DXT1_EXT,
+  COMPRESSED_SRGB_ALPHA_S3TC_DXT3_EXT,
+  COMPRESSED_SRGB_ALPHA_S3TC_DXT5_EXT,
+} = GL_EXTENSIONS_CONSTANTS;
+
+// eslint-disable-next-line complexity
+const isFormatSupported = (gl, format) => {
```

Comment:
Move functions to utils. I think `src\utils\debug\texture-render-utils.ts` is a right place